thoth/storages/graph/postgres.py

```diff
@@ -3429,8 +3429,8 @@ def sync_solver_result(self, document: dict) -> None:
             for error_info in document["result"]["errors"]:
                 # Normalized in `_create_python_package_version'.
                 package_name = error_info.get("package_name") or error_info["package"]
-                package_version = error_info["version"]
-                index_url = error_info["index"]
+                package_version = error_info.get("package_version")
+                index_url = error_info.get("index_url")
```

Comment:
With this, we will not be consistent with older versions. Can this be switched to something like this?

```python
                package_version = error_info.get("package_version", error_info["version"])
                index_url = error_info.get("index_url", error_info["index"])
```

Reply:
Sure, thanks!